validator/server_arb/validator_spawner.go

```diff
@@ -105,7 +105,7 @@ func (s *ArbitratorSpawner) WasmModuleRoots() ([]common.Hash, error) {
 }
 
 func (s *ArbitratorSpawner) StylusArchs() []rawdb.WasmTarget {
-	return []rawdb.WasmTarget{rawdb.TargetWavm}
+	return []rawdb.WasmTarget{rawdb.TargetWavm, rawdb.LocalTarget()}
```

Comment:
why is that needed?

Reply:
Non jit tests, such as TestProgramInfiniteLoopShouldCauseErrOutOfGas, fail without this rawdb.LocalTarget():
``` 
panic: failed to get compiled program for activated program, program: 0x457b1BA688E9854BDbed2f473F7510C476A3dA09, local target missing: true, err: <nil> [recovered]
```
 So I supposed this local target was missing here :thinking:

Reply:
o.k. so instead what we should do is make sure that when recording we also collect for localTarget.
It can either be done inside NewMessageRecordingContext (and in the same time also NewMessageCommitContext)
or in (c *MessageRunContext) WasmTargets()
or in block_recorder, when we call NewMessageRecordingContext (we already make sure loaclTarget is included for CommitContext).
I kind of tend for the last option, but all seem reasonable to me.